.circleci/config.yml

```diff
@@ -481,12 +481,19 @@ jobs:
       - run:
           name: Install Cypress dependencies
           command: |
-            ${SUDO} apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
+            ${SUDO} apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 jq curl
 
       - run:
-          name: Init credentials
+          name: Init config
           command: |
             mkdir ~/.streamlit
+            MAPBOX_TOKEN=$(curl -sS https://data.streamlit.io/tokens.json | jq -r '.mapbox')
+            echo '[mapbox]' >  ~/.streamlit/config.toml
+            echo 'token = "'$MAPBOX_TOKEN'"' >> ~/.streamlit/config.toml
```

Comment:
We need a solution for local e2e tests too

Reply:
I added the same for the dockerfile